services/net/cpnews/appsettings.Staging.json

```diff
@@ -0,0 +1,26 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "Service": {
+    "MaxFailLimit": 5,
+    "ApiUrl": "http://api:8080",
+    "MediaTypes": ["CPNews"]
+  },
+  "Auth": {
+    "Keycloak": {
```

Comment:
Checkout the latest changes in my prior PR for configuration. There are some differences in the Keycloak section.

Reply:
Just remove the "Client" property

Reply:
Removed it.